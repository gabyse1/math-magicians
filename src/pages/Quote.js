const Quote = () => (
  <main>
    <h3>
      Mathematic is about numbers, equations, computations, or algorithms:
      It is about understanding. -William Paul Thurston
    </h3>
  </main>
);

export default Quote;
