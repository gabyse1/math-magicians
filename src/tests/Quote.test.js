import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

describe('<Quote /> component', () => {
  it('Component render correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
