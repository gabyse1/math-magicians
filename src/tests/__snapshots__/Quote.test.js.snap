// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Quote /> component Component render correctly 1`] = `
<main
  className="page__quote"
>
  <p
    className="quote__message"
  >
    "
    Mathematic is about numbers, equations, computations, or algorithms: It is about understanding.
    "
  </p>
  <p
    className="quote__author"
  >
    —
    <strong>
      William Paul Thurston
    </strong>
    —
  </p>
</main>
`;
